### PR TITLE
Feature display resource administrate

### PR DESCRIPTION
**Pull request by nikos83**

### CODE REVIEW

File: app/dashboards/car_dashboard.rb [hunk 1]
```diff
@@ -62,8 +62,11 @@ class CarDashboard < Administrate::BaseDashboard
 
   # Overwrite this method to customize how cars are displayed
   # across all pages of the admin dashboard.
-  #
   # def display_resource(car)
```
Comment (fab-ian) [inline app/dashboards/car_dashboard.rb]: Remove this type of comments from all Dashboards. It's look strange :) In general we shouldn't leave unnecessary comments. We will talk about it in a Hangout meeting.

File: app/dashboards/car_dashboard.rb [hunk 2]
```diff
@@ -62,8 +62,11 @@ class CarDashboard < Administrate::BaseDashboard
...
   #   "Car ##{car.id}"
   # end
+
+  def display_resource(car)
+    car.registration_number
```
Comment (fab-ian) [inline app/dashboards/car_dashboard.rb]: I think we should use a little more complex form for this label. Maybe "Brand Model: Registration number". For example: Honda Accord: WX 2345. If someone has many cars using only 'registration number' can be confusing.

File: app/dashboards/car_dashboard.rb [hunk 3]
```diff
@@ -62,8 +62,11 @@ class CarDashboard < Administrate::BaseDashboard
 
   # Overwrite this method to customize how cars are displayed
   # across all pages of the admin dashboard.
-  #
   # def display_resource(car)
   #   "Car ##{car.id}"
   # end
+
+  def display_resource(car)
+    car.brand + ' ' + car.model + ' ' + car.registration_number
```
Comment (fab-ian) [inline app/dashboards/car_dashboard.rb]: Please use the Ruby way.
``` "#{car.brand} #{car.model}: #{car.registration_number}" ```

Comment (nikos83) [reply app/dashboards/car_dashboard.rb]: I'll change it for all of dashboards

**Pull request merged**